admin_manual/configuration_files/external_storage_configuration_gui.rst

```diff
@@ -19,6 +19,10 @@ The External storage support application is enabled on your Apps page.
 Storage configuration
 ---------------------
 
+To access the settings for configuring external storage mounts, click on your Profile icon
```

Comment:
To access the settings to configure external storage mounts, click on your profile icon